src/datalinker/handlers/external.py

```diff
@@ -171,9 +175,14 @@ async def timeseries(
     # In those cases we have to join with another table on ccdVisitId.
     if join_time_column:
         join_table, time_column = join_time_column.rsplit(".", 1)
+        join_clause: str
```

Comment:
No type declaration is needed here (mypy can determine this automatically) and therefore none should be given.